sigmf-spec.md

```diff
@@ -200,10 +200,15 @@ Rules for SigMF Collection files:
 Rules for SigMF Archive files:
 1. The Archive MUST use the `tar` archive format, as specified by POSIX.1-2001.
 1. The Archive file's filename extension MUST be `.sigmf`.
-1. The Archive MUST contain the following files: for each contained Recording
-   with some name given here meta-syntactically as `N`, files named `N` (a
-   directory), `N/N.sigmf-meta`, and `N/N.sigmf-data`.
-1. The Archive MAY contain a `.sigmf-collection` file in the top-level directory.
+1. The Archive MUST contain at least one SigMF Recording.
+1. The Archive MAY organize SigMF Recordings into subdirectories such that for
+   each contained Recording with some base name `N`, individual files are stored
+   within the directory `N` (e.g.: `N/N.sigmf-meta`, and `N/N.sigmf-data`). The
+   use of subdirectories within an archive is NOT RECOMMENDED and may be
```

Comment:
Why NOT RECOMMENDED? Why not just let users be free to use whatever *internal* directory structure they want to? Seems to me that (from the application side) for `subdir1/subdir2/a_recording.sigmf-{meta,data}` treating the entire `subdir1/subdir2/a_recording` section as the recording "name" will be adequate for most purposes: opening files (after extraction), identifying contents within unextracted archives, and adding files to new archives.

Reply:
See above. Users are still free to manage Recordings, Collections, and SigMF Archives however they please and can use tar archives or compressed archives to store data outside of what is explicitly codified by this specification. The main purpose of canonical SigMF Archives is to specify a simple, single, directly usable file (that should generally contain either one Recording or one Collection). To that end they should be as straightforward as possible.

Reply:
Entirely removed in favor of a more permissive rule.